mmv1/products/compute/InterconnectAttachment.yaml

```diff
@@ -312,3 +312,15 @@ properties:
       - :IPV4_IPV6
       - :IPV4_ONLY
     default_from_api: true
+  - !ruby/object:Api::Type::String
+    name: 'cloudRouterIpv6Address'
```

Comment:
Would it be feasible to add a test that's IPV4_IPV6? If there's a significant setup cost we are fine without it, but both current tests for this resource are IPV4_ONLY.

Reply:
The attribute added here isn't easily testable for the reasons noted in the PR: https://github.com/GoogleCloudPlatform/magic-modules/pull/9890

While I didn't check whether these output-only attributes are present without that, my guess is that without setting the `stack_type` to be both, the outputs of these attributes will be empty, so there's not much point in testing them without having the `stack_type` attribute set to support ipv6.

Reply:
That tracks, thanks!